addons/default/visiosoft/advs-module/resources/js/filter_modal.js

```diff
@@ -141,4 +141,13 @@ $('.set_category').on('click', function () {
 
     $('#filterModal').find('form').attr("action", '/advs/list');
     $('#filterModal').modal('toggle');
-})
+});
+
+// Move filter on small screen
+$(window).resize(function() {
+    const width = (window.innerWidth > 0) ? window.innerWidth : screen.width;
+    if (width <= 575) {
```

Comment:
is it best practice to do like this? @diashalabi @ozcan-durak

Reply:
i think its better than including the same file twice, once for the mobile version and once for the desktop version + including it twice is causing warnings in the console, you can check the issue I commented above @fatihalp

Reply:
Yeah thats ok i think

Reply:
isnt possible to use media? @ozcan-durak @diashalabi

Reply:
the old way was using media, but since we have two of the same inputs with the same ids we were getting dublicated ids warning, I shared a screen shot on the issue https://gitlab.com/openclassify/openclassify/-/issues/1619 @fatihalp